lib/PuppeteerSharp/ChromiumProcess.cs

```diff
@@ -16,7 +11,7 @@ namespace PuppeteerSharp
     /// Represents a Chromium process and any associated temporary user data directory that have created
     /// by Puppeteer and therefore must be cleaned up when no longer needed.
     /// </summary>
-    public class ChromiumProcess : IDisposable
+    public class ChromiumProcess : ProcessBase
```

Comment:
What do you think about calling `ProcessBase` `LauncherBase` and calling these two `ChromiumLauncher` and `FirefoxLauncher`?

Reply:
@kblok done